website/docs/components/tooltip/partials/code/how-to-use.md

```diff
@@ -0,0 +1,168 @@
+<Hds::TooltipButton @text="Hello!" @placement="right">
+  <Hds::Badge @text="Some tests failed" @icon="alert-triangle" @color="warning" />
+</Hds::TooltipButton>
+```
+
+#### Offset
+
+You can change the offset of the tooltip in relation to the opener element content if needed.
+
+```handlebars
+<Hds::TooltipButton @text="Hello!" @offset={{array 50 30}}>
+  <Hds::Stepper::Step::Indicator @text="1" @status="incomplete" />
+</Hds::TooltipButton>
+```
+
+### isInline
+
+You can change the default `inline-block` display to use `flex` display if needed.
+
+```handlebars
+<p>
+  Text before
+  <Hds::TooltipButton 
+    @text="Hello!" 
+    aria-label="HashiCorp" 
+    @isInline={{false}}
+  >
+    <FlightIcon @name="hashicorp" />
+  </Hds::TooltipButton>
+  Text after
+</p>
+```
+
+#### Extra Tippy Options
+
+You can use `@extraTippyOptions` to provide more specific options to [Tippy.js](https://atomiks.github.io/tippyjs/). For a full list of available options refer to the [Tippy.js API documentation](https://atomiks.github.io/tippyjs/v6/all-props/).
+
+For example, you can use the `allowHTML` option to enable rich tooltip text:
+
+```handlebars
+<Hds::TooltipButton
+  @extraTippyOptions={{hash allowHTML=true}}
+  @text="<b>Hello</b> <em>there</em>!"
+>
+  More information
+</Hds::TooltipButton>
+```
+
+!!! Warning
+
+If you enable the `allowHTML` option:
+- be sure to sanitize your data
+- to maintain accessibility, do not include interactive content such as links or buttons
+- we recommend using only basic inline-level text formatting tags such as `strong` or `em`; using block-level tags such as `div` or `p` will make the HTML syntax invalid
+
+!!!
+
+### Ember modifier
+
+An Ember modifier is available if your use case requires attaching a tooltip to an element than is already interactive (to be accessible, tooltips should only be attached to interactive elements, like buttons, links, inputs, etc).
+
+#### Modifier used on a link
+
+```handlebars
+<Hds::Link::Standalone 
+  {{hds-tooltip "Hello!"}} 
+  @href="#" 
+  @icon="collections" 
+  @text="Read tutorial" 
+/>
+```
+
+#### Placement
+```handlebars
+<Hds::Button 
+  {{hds-tooltip "Hello!" 
+  options=(hash placement="right")}} 
+  @icon="external-link" 
+  @text="Visit website" 
+  @href="https://hashicorp.com" 
+/>
+```
+
+#### Offset
+```handlebars
+<Hds::Form::Field @layout="vertical" as |F|>
+  <F.Label @controlId="tooltip-example-control-id">First Name</F.Label>
+  <F.Control>
+    <Hds::Form::TextInput::Base 
+      {{hds-tooltip "Hello!" 
+      options=(hash placement="top" 
+      offset=(array 0 30))}} 
+      @type="text" 
+      @value="Jane" 
+      id="tooltip-example-control-id"
+      @width="200px" 
+    />
+  </F.Control>
+</Hds::Form::Field>
+```
+
+#### Extra Tippy Options
+
+You can enable extra [Tippy.js options](https://atomiks.github.io/tippyjs/v6/all-props/) by passing a hash of the options you wish to use similarly to how the `TooltipButton` component works.
```

Comment:
I wonder if this would be the right place to caution consumers about the potential non-conformance of some tippy options.

```suggestion
While you can enable extra [Tippy.js options](https://atomiks.github.io/tippyjs/v6/all-props/) by passing a hash of the options you wish to use similarly to how the `TooltipButton` component works, you should make sure that any additional options chosen will still result in a WCAG-conformant outcome.
```

Reply:
Yes, but I would not start the paragraph with `While you can`.  We want to convey that they **can**. And then specify that in that case they have to make sure the result is WCAG-conformant.

Reply:
How about:

You can enable extra [Tippy.js options](https://atomiks.github.io/tippyjs/v6/all-props/) by passing a hash of the options you wish to use similarly to how the `TooltipButton` component works. You should make sure that any additional options chosen will still result in a WCAG-conformant outcome.

Reply:
I went ahead and updated the text as I suggested above. Please open another comment if you think it still needs changing.